module-sys/SystemManager/PowerManager.cpp

```diff
@@ -187,10 +187,11 @@ namespace sys
             level = bsp::CpuFrequencyMHz::Level_2;
             break;
         case bsp::CpuFrequencyMHz::Level_2:
-            level = powerProfile.minimalFrequency;
+            level = bsp::CpuFrequencyMHz::Level_1;
```

Comment:
This adds intermediate step of 12MHz when decreasing frequency. Is there any argument for it?

Reply:
The goal was to go as fast as possible to the lowest frequency. For delaying the frequency steps the counter values in power profiles should be used. This change will affect Bell.